buildozer/default.spec

```diff
@@ -58,7 +58,12 @@ requirements = python3,kivy
 # Valid options are: landscape, portrait, portrait-reverse or landscape-reverse
 orientation = portrait
 
-# (list) List of service to declare
+# (list) List of services to declare
+# Each service consists of a name (a valid Java class name, with the first letter capitalized)
+# followed by a colon, followed by the name of the Python script (.py file) that should be
+# launched. This is optionally followed by ":foreground" for foreground services or
+# ":foreground:sticky" for sticky foreground services. The default is a background service.
+# Bound services are not supported.
```

Comment:
Maybe we want to say that (at least ATM) this is an android-specific feature, even if does not have the `android.*` prefix?

Reply:
Ah, I hadn't even noticed it didn't have an android prefix. I am on it.